### PR TITLE
feat(MessageSend): add flags

### PR DESCRIPTION
**Pull request by ozraru**
https://discord.com/developers/docs/resources/channel#create-message

### CODE REVIEW

File: message.go [hunk 1]
```diff
@@ -234,6 +234,7 @@ type MessageSend struct {
 	AllowedMentions *MessageAllowedMentions `json:"allowed_mentions,omitempty"`
 	Reference       *MessageReference       `json:"message_reference,omitempty"`
 	StickerIDs      []string                `json:"sticker_ids"`
+	Flags           int                     `json:"flags,omitempty"`
```
Comment (Earlopain) [inline message.go:237]: Feels to me like this should be the flag type that  describes the possible values in the other PR. I know there are places that do it like this as well but that seems like legacy @FedorLap2006 ?

**Review by FedorLap2006 — CHANGES_REQUESTED** [review]
As @Earlopain mentioned, `Flags` should have `MessageFlags` type instead of `int`.

File: message.go (hunk 1 above)
Comment (ozraru) [reply message.go]: Fixed. Sorry for my rudimentary mistake.

**Review by FedorLap2006 — APPROVED** [review]

**Comment by FedorLap2006** [comment]
Thanks for your contribution!

**Pull request merged**